### ISSUE TITLE
mfscli display realsize bug

### ISSUE THREAD

**Issue by tclh123** [moosefs/moosefs#100]
There's one path in my cluster, display `EB` in mfsdirinfo `realsize` output.
I think this is a bug.

```
$  mfsdirinfo -H /mfs/xxx/yyy
/mfs/xxx/yyy:
 inodes:         35k
  directories:   185
  files:         35k
 chunks:         23k
 length:       4.0GB
 size:         5.7GB
 realsize:      18EB

$ mfscli -v
version: 3.0.92

$ /usr/sbin/mfsmaster -v
version: 3.0.92-1
```

**Comment by oxide94** [comment]
Hi @tclh123,
It is fixed in MooseFS 3.0.101, we will release it soon.

Thanks,
Best regards,
Peter / MooseFS Team

**Comment by acid-maker** [comment]
Fixed by commit bb83282f03541bc99603bcc41135dafbb8c65cb6
